README.md

```diff
@@ -153,10 +153,12 @@ Inspired by [awesome-python](https://github.com/vinta/awesome-python).
 - [Technaureus](https://www.youtube.com/channel/UChusfuYvl4Xbf7-RLP5hHFg) - English channel
 - [Odoo Bangla](https://www.youtube.com/channel/UCoVnWNoF0kQbukKqN_DxxfQ) - Bangla channel
 - [Luigys Toro](https://www.youtube.com/channel/UCAoYNRiCdqVGxAbNoiUODRg) - Spanish channel
+- [Nicholas Kosinski](https://www.youtube.com/channel/UCfmxKTk4iO2PSdSVOLKhwBA) - English channel
 
 ## Courses
 
-- [Official Odoo Courses](https://www.odoo.com/es_ES/slides/all) - Official Courses
+- [Official Odoo Courses] (https://www.odoo.com/slides) - Official Courses - English
+- [Official Odoo Courses](https://www.odoo.com/es_ES/slides/all) - Official Courses - Spanish
```

Comment:
This link points to an English version

Reply:
https://www.odoo.com/es_ES/slides/all shows the User Menus and other UI features (navigation particularly) in Spanish.

https://www.odoo.com/slides shows the User Menus and other UI features (navigation particularly) in English.

Video content is in English for all courses.